docs-compose/overwatch_docs/content/DataEngineer/Definitions/071x.md

```diff
@@ -0,0 +1,1005 @@
+| service_name              | string  | Name of the service corresponding to DBSQL warehouse                                                                                                                                                                      |                                                                                                                   
+| action_name               | string  | create, edit, or snapImpute – depicts the type of action for the cluster – **snapImpute is used on first run to initialize the state of the cluster even if it wasn’t created/edited since audit logs began               |              
+| user_email                | string  | Email of the user requesting the action                                                                                                                                                                                   |                                                                                                                                                                                  
+| cluster_size              | string  | Size of the clusters allocated for this warehouse. Increasing the size of a spark cluster allows you to run larger queries on it. If you want to increase the number of concurrent queries, please tune max_num_clusters  | 
+| min_num_clusters          | long    | Minimum number of available clusters that will be maintained for this SQL warehouse.                                                                                                                                      | 
+| max_num_clusters          | long    | Maximum number of clusters that the autoscaler will create to handle concurrent querie                                                                                                                                    | 
+| auto_stop_mins            | long    | The amount of time in minutes that a SQL warehouse must be idle (i.e., no RUNNING queries) before it is automatically stopped.                                                                                            | 
+| spot_instance_policy      | string  | Configurations whether the warehouse should use spot instances.                                                                                                                                                           | 
+| enable_photon             | boolean | Configures whether the warehouse should use Photon optimized clusters.                                                                                                                                                    | 
+| channel                   | struct  | This column contains channel details. Some examples - CHANNEL_NAME_UNSPECIFIED, CHANNEL_NAME_PREVIEW, CHANNEL_NAME_CURRENT                                                                                                |
```

Comment:
If there are more options here, where can they read more about it? Can we link to docs?

Reply:
I have fetched these values from API docs. Let me know if we can add the API docs link here